de.blobbyvolley.BlobbyVolley2.yaml

```diff
@@ -0,0 +1,64 @@
+app-id: de.blobbyvolley.BlobbyVolley2
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: blobby
+
+finish-args:
+  - --device=dri
+  - --socket=x11
```

Comment:
It uses SDL2; does it work natively on Wayland?

Reply:
It does work natively on Wayland (I didn't check it before) but Blobby Volley running natively on Wayland does not have window decorations: user is not able to move window without knowing modifier key functionality which is very bad UX. This game is mostly played in window mode, not full screen so this matters.